examples/screenshare/main.go

```diff
@@ -8,6 +8,8 @@ import (
 	_ "github.com/pion/mediadevices/pkg/codec/openh264" // This is required to register h264 video encoder
 	_ "github.com/pion/mediadevices/pkg/codec/opus"     // This is required to register opus audio encoder
 	_ "github.com/pion/mediadevices/pkg/codec/vpx"
```

Comment:
Sorry, I had forgotten to add a comment to vpx import. Would you add it?

Reply:
Yep, I'll add it